boot/boot.ks

```diff
@@ -53,15 +53,20 @@ ELSE {
 	}
 	ELSE {
 		PRINT "No existing RAMP files detected. Trying to raise antennas and rebooting...".
-		FOR P IN SHIP:PARTS {
-			IF P:MODULES:CONTAINS("ModuleDeployableAntenna") {
-				LOCAL M IS P:GETMODULE("ModuleDeployableAntenna").
-				FOR A IN M:ALLACTIONNAMES() {
-					IF A:CONTAINS("Extend") { M:DOACTION(A,True). }
-				}.
-			}
-		}.
-		REBOOT.
+		IF Career():CANDOACTIONS {
```

Comment:
Could simplify the code by calling `partsExtendAntennas` here instead of duplicating the for loops and the can-do checking logic. @fellipec, any downside to running `lib_parts` from inside the boot script?

Reply:
Unfortunatelly yes. The boot script still have not a copy of lib_parts (or any other library) on the ship, and if can't have a home connection it also can't switch to archive to run it. 

I'll try to test those files in next days, but they look fine.